pkgs/mime/tool/update_media_types.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:http/http.dart' as http;
+
+/// Define additions to the base apache/httpd mime set.
+///
+/// These are deltas to the base set that make sense in a Dart context
+/// (programming environments, Dart implemented web servers, ...).
+const Map<String, List<String>> mimeAdditions = {
+  'text/markdown': ['md', 'markdown'],
+  'text/x-dart': ['dart'],
+};
+
+void main() async {
+  // https://github.com/apache/httpd/blob/trunk/docs/conf/mime.types
+  const mimeTypesUrl =
+      'https://raw.githubusercontent.com/apache/httpd/refs/heads/trunk/docs/conf/mime.types';
+
+  print('Reading from $mimeTypesUrl ...');
+  final response = await http.get(Uri.parse(mimeTypesUrl));
+
+  final lines = response.body
+      .split('\n')
+      .where((line) => !line.startsWith('#'))
+      .map((line) => line.trim())
+      .where((line) => line.isNotEmpty);
+
+  final extensionToMime = <String, String>{};
+  final mimeToExtensions = <String, List<String>>{};
+
+  final wsRegex = RegExp('  +');
+  for (var line in lines) {
+    line = line.replaceAll('\t', ' ').replaceAll(wsRegex, ' ');
+
+    final segments = line.split(' ');
```

Comment:
And we can combine with this too, gettting `final segments = line.split(RegExp(r'[\t ]+'));`